### DISCUSSION TITLE
Modification of the situation impossible with the keyboard

### DISCUSSION THREAD

**Discussion by freemans32** [Stellarium/stellarium#2305] [Q&A]
Hello everyone, I am trying to modify the coordinates of the observation site but cannot modify them using the keyboard.
I managed to modify the degrees, munites and the tens of seconds but not after the comma.
Do you have a solution for me?

Thank you in advance.

**Comment by alex-w** [comment]
Please check https://github.com/Stellarium/stellarium-data/releases/tag/weekly-snapshot
